WooCommerce/Classes/ViewRelated/Text View Screen/TextViewViewController.swift

```diff
@@ -1,5 +1,14 @@
 import UIKit
 
+protocol TextViewViewControllerDelegate: class {
+
+    // Text validation.
+    func shouldValidate(text: String) -> Bool
+
+    // Error text that will be shown if the text does not pass validation.
+    func validationError() -> String
```

Comment:
nit suggestion: from the function name, it sounds like it's returning an `Error` to me - not sure if `validationErrorMessage`, or combining the error message with the validation result in the function above would be more clear?